src/handlebars/about.handlebars

```diff
@@ -7,53 +7,81 @@
 
     <div id="support-container" class="blue-link-section">
 
-      <!-- TODO this might be renamed to GetOpenJDK to avoid confusion -->
       <div class="margin-bottom">
-        <h2 class="bold">About AdoptOpenJDK - GetOpenJDK</h2>
+        <h2 class="bold">About AdoptOpenJDK.net</h2>
         <p>
```

Comment:
Given the main title appearing immediately ahead of this section is "About", how about we call it "Background"

Reply:
Fixed